int64 num_of_fingers
---
sq_grasping/graspArr grasps
